js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeRequestedPartitionsLink.tsx

```diff
@@ -32,9 +34,10 @@ import {
 interface Props {
   runIds?: string[];
   partitionKeys: string[];
+  intent?: React.ComponentProps<typeof BlueprintTag>['intent'];
```

Comment:
You should be able to reference the `@dagster-io/ui` component instead of pulling in the underlying Blueprint import, e.g.

```
import {Tag} from '@dagster-io/ui`;

// then
intent?: React.ComponentProps<typeof Tag>['intent'];
```